games/p.yaml

```diff
@@ -949,19 +949,19 @@
   type: remake
   updated: 2015-06-01
 
-- name: Pushover
+- name: Domino-Chain
```

Comment:
please move this game to `d.yaml`

Reply:
Of course! 🤦
That's what happens when doing stuff in the middle of the night.
I've pushed a fix.